GNUmakefile.in

```diff
@@ -48,14 +56,14 @@ check: $(tests) | all
 
 .PHONY: clean
 clean:
-	rm -f $(commands) $(testdir)/*.err
+	rm -f $(commands) $(testdir)/*.err $(addprefix $(man1dir)/,$(man1pages)) $(addsuffix .rst,$(addprefix $(man1dir)/,$(man1pages)))
```

Comment:
The following seems to delete the installed files 
```
make clean
rm -f sumaxy tests/*.err /usr/local/share/man/man1/sumaxy.1 /usr/local/share/man/man1/sumaxy.1.rst
```
while `make clean` should clean the tmp/build directory. You might want to use this code for `make uninstall` and try the following for `make clean`
```
rm -f $(commands) $(testdir)/*.err $(manpages) $(addsuffix .rst,$(manpages))
```
@mimi1vx you were a bit fast.